tmk_core/common/chibios/pin_defs.h

```diff
@@ -53,6 +53,9 @@
 #        define D5 PAL_LINE(GPIOC, 13)
 #        define B0 PAL_LINE(GPIOC, 14)
 #    endif
+#elif defined(KEYBOARD_kinesis_kint36)
+// The compatibility defines do not work with the KINETIS header files,
+// so do not declare anything.
```

Comment:
```suggestion
```

Reply:
See the other comment on the same line below regarding define availability